galgebra/mv.py

```diff
@@ -1162,25 +1159,19 @@ def Fmt(self, fmt=1, title=None):
         with one grade per line.  Works for both standard printing and
         for latex.
         """
-        if printer.GaLatexPrinter.latex_flg:
-            printer.GaLatexPrinter.prev_fmt = printer.GaLatexPrinter.fmt
-            printer.GaLatexPrinter.fmt = fmt
-        else:
-            printer.GaPrinter.prev_fmt = printer.GaPrinter.fmt
-            printer.GaPrinter.fmt = fmt
-
         if title is not None:
             self.title = title
 
         if printer.isinteractive():
             return self
 
         if printer.GaLatexPrinter.latex_flg:
-            s = printer.GaLatexPrinter().doprint(self)
+            p = printer.GaLatexPrinter()
         else:
-            s = printer.GaPrinter().doprint(self)
+            p = printer.GaPrinter()
+        p.fmt = fmt
+        s = p.doprint(self)
```

Comment:
This approach did not work for interactive mode, as `fmt` would be ignored. This is addressed by gh-369